File: src/com/gb/javalearn/lesson9final/CSharp/Lesson9_task66.cs
using System;

namespace SumNaturalNumbersRecursion
{
    class Program
    {
        private static void Main(string[] args)
        {
            Console.Write("Введите число m: ");
            int m = int.Parse(Console.ReadLine());
            Console.Write("Введите число n: ");
            int n = int.Parse(Console.ReadLine());
            int sum = SumNaturalNumbers(m, n);
            Console.WriteLine($"Сумма натуральных чисел от {m} до {n}: {sum}");
        }

        private int SumNaturalNumbers(int m, int n)
        {
            if (m <= n)
                return m + SumNaturalNumbers(m + 1, n);
            return 0;
        }
    }
}

File: src/com/gb/javalearn/lesson9final/Lesson9task66.java
package com.gb.javalearn.lesson9final;

import java.util.Scanner;

/*
 Задача 66: Задайте значения M и N. Напишите программу, которая найдёт сумму натуральных элементов в промежутке от M до N.
 Task 66: Set the values of M and N. Write a program that will find the sum of positive integers between M and N.
 */
public class Lesson9task66 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Введите число m: ");
        int m = scanner.nextInt();
        System.out.print("Введите число n: ");
        int n = scanner.nextInt();
        int sum = sumNaturalNumbers(m, n);
        System.out.printf("Сумма натуральных чисел от %d до %d: %d\n", m, n, sum);
    }

    public static int sumNaturalNumbers(int m, int n) {
        if (m > n) {
            return 0;
        } else {
            return m + sumNaturalNumbers(m + 1, n);
        }
    }
}
